docs/content/en/integrations/parsers/file/blackduck_binary_analysis.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Blackduck Binary Analysis"
+toc_hide: true
+---
+2 options:
+
+* Import a single BDBA vulnerabilty csv results file.
```

Comment:
What is the other option? or do you want to delete '2 options'? Could you also provide a link to the scanner?

Reply:
Good catch - Docs updated.  Thanks again @manuel-sommer